cookiecutter.json

```diff
@@ -0,0 +1,78 @@
+{
+    "project_name": "infection-monkey-template",
+
+    "full_name": "Your Name",
+    "email": "you@gmail.com",
+    "github_repo": "infection-monkey-template",
+
+    "platform": [
+        "github",
+        "gitlab"
+    ],
+    
+    "org": "guardicode",
+    "github_url": "guardicode",
+    "url": "https://{{ cookiecutter.platform }}.com/{{ cookiecutter.org }}/{{ cookiecutter.project_name }}",
+
+    "__year": "{% now 'utc', '%Y' %}",
+    "license": ["MIT","Apache","GPLv3"],
+    
+    "default_branch": "main",
+    "project_short_description": "Sample cookiecutter template for infection monkey.",
+    "python_major_version": 3,
+    "python_minor_version": 11,
+
+    "hook_isort": ["y", "n"],
+    "hook_black": ["y", "n"],
+    "hook_vulture": ["y", "n"],
+    "hook_flake8": ["y", "n"],
+    "hook_eslint": ["y","n"],
+    "hook_mypy": ["y","n"],
+    "hook_shellcheck": ["y","n"],
+
+    "codecov": ["y","n"],
+    
+    "exclude_these_file_in_flake8": "monkey/monkey_island/cc/ui,vulture_allowlist.py",
```

Comment:
This should be variable.

Reply:
This is the default value defined in the cookiecutter.json file for this variable. If the user doesn't provide any value for this field then it would be set as default otherwise the user input would only be set. 

if you want I can provide this `"monkey/monkey_island/cc/ui,vulture_allowlist.py"` in the prompt while filling out the value for this variable and in that, if the user doesn't provide any option then the field would be set empty.

Reply:
We can drop this as an option. The user can fill it in later if they want.